2024/shorts/cube-shadow-puzzle/french/sentence_translations.json

```diff
@@ -24,23 +24,23 @@
   ]
  },
  {
-  "translatedText": "Ou formulé différemment, si vous répétez ce processus encore et encore, en lançant ce cube au hasard et en notant les aires que vous trouvez, vers quelle valeur tendrait la moyenne empirique de toutes ces aires lorsque vous le faites de plus en plus ?",
+  "translatedText": "Ou formulé différemment, si vous répétez ce processus encore et encore, en tournant ce cube au hasard et en notant les aires que vous trouvez, vers quelle valeur tendrait la moyenne empirique de toutes ces aires lorsque vous le faites de plus en plus?",
   "input": "Or phrased another way, if you repeat this process over and over, randomly tossing that cube and tallying up the areas that you find, what would the empirically measured mean of all those measured areas approach as you do this more and more?",
   "time_range": [
    10.7,
    22.64
   ]
  },
  {
-  "translatedText": "À proprement parler, en faisant référence à l'ombre, cela dépend de l'endroit où se trouve la source de lumière, mais ici nous faisons ce que la plupart des mathématiciens font : nous considérons la source de lumière comme étant infiniment éloignée, donc l'ombre n'est qu'une projection plate, disons, sur le plan xy.",
+  "translatedText": "Pour être franc, en faisant référence à l'ombre, cela dépend de l'endroit où se trouve la source de lumière, mais ici nous faisons ce que la plupart des mathématiciens font: nous considérons la source de lumière comme étant infiniment éloignée, donc l'ombre n'est qu'une projection plate, disons, sur le plan x y.",
   "input": "Strictly speaking, in referencing the shadow, that would depend on where the light source is, but here we're doing the typical mathematician thing of thinking of the light source as being infinitely far away, so the shadow is just a flat projection, say, onto the xy-plane.",
   "time_range": [
    23.24,
    35.66
   ]
  },
  {
-  "translatedText": "C'est assurément un problème difficile, et je suis curieux de savoir comment vous l'aborderiez dans les commentaires.",
+  "translatedText": "C'est assurément un problème difficile, et je suis curieux de savoir comment vous le résoudriez dans les commentaires.",
```

Comment:
"Aborder" était ici aussi la traduction adéquate de "Approach". Sinon, c'est "how you would solve" qui aurait été utilisé.